### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by isahwahni**
no changes in the code
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ibatta/SelFW/pull/19%23discussion_r160903282%22%2C%20%22https%3A//github.com/ibatta/SelFW/pull/19%23discussion_r160903510%22%2C%20%22https%3A//github.com/ibatta/SelFW/pull/19%23issuecomment-356882872%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%2055f43009789c7d9c30d627f591d139af742626ae%20src/com/generic/setup/PagesURLs.java%206%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ibatta/SelFW/pull/19%23discussion_r160903510%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Please%20remove%20this%20if%20it%20is%20not%20used%20%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222018-01-11T09%3A37%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/18301319%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ibatta%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/com/generic/setup/PagesURLs.java%3AL1-223%22%7D%2C%20%22Pull%2055f43009789c7d9c30d627f591d139af742626ae%20src/com/generic/setup/PagesURLs.java%2024%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ibatta/SelFW/pull/19%23discussion_r160903282%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Please%20overload%20the%20get%20function%20below%20to%20make%20it%20take%20parameter%20String%20driverName%2C%20it%20should%20use%20getDriverpath%28%29%3B%20where%20it%20should%20return%20in%20this%20case%20%20the%20full%20driver%20path%5Cr%5Cnc%3A/...../chromeDriver.exe%22%2C%20%22created_at%22%3A%20%222018-01-11T09%3A36%3A00Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/18301319%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ibatta%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/com/generic/setup/PagesURLs.java%3AL1-223%22%7D%2C%20%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ibatta/SelFW/pull/19%23issuecomment-356882872%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40ibatta%20%5Cr%5CnThe%20requested%20changes%20are%20done%20in%20the%20following%20commit%20%3A%20%5Cr%5Cnhttps%3A//github.com/ibatta/SelFW/pull/19/commits/050fa1abbacd53837a1f40be21b83442892b4421%5Cr%5Cn%5Cr%5CnThanks%22%2C%20%22created_at%22%3A%20%222018-01-11T09%3A52%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/33652900%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/isahwahni%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull 55f43009789c7d9c30d627f591d139af742626ae src/com/generic/setup/PagesURLs.java 24'></a> &nbsp;<b><a href='https://github.com/ibatta/SelFW/pull/19#discussion_r160903282'>File: src/com/generic/setup/PagesURLs.java:L1-223</a></b>
- <a href='https://github.com/ibatta'></a> Please overload the get function below to make it take parameter String driverName, it should use getDriverpath(); where it should return in this case  the full driver path
c:/...../chromeDriver.exe
- [x] <a href='#crh-comment-Pull 55f43009789c7d9c30d627f591d139af742626ae src/com/generic/setup/PagesURLs.java 6'></a> &nbsp;<b><a href='https://github.com/ibatta/SelFW/pull/19#discussion_r160903510'>File: src/com/generic/setup/PagesURLs.java:L1-223</a></b>
- <a href='https://github.com/ibatta'></a> Please remove this if it is not used
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/ibatta/SelFW/pull/19#issuecomment-356882872'>General Comment</a></b>
- <a href='https://github.com/isahwahni'></a> @ibatta
The requested changes are done in the following commit :
https://github.com/ibatta/SelFW/pull/19/commits/050fa1abbacd53837a1f40be21b83442892b4421
Thanks

<a href='https://www.codereviewhub.com/ibatta/SelFW/pull/19?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ibatta/SelFW/pull/19?approve=1'></a>&nbsp;<a href='https://github.com/ibatta/SelFW/pull/19'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ibatta** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ibatta/SelFW/pull/19'>Refresh</a>.<a href='#crh-update-none'></a>

File: src/com/generic/setup/PagesURLs.java [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+	public static String addressBookPage = "addressBookPage";
+	public static String savedCartsPage = "savedCartsPage";
+	public static String consentManagementPage = "consentManagementPage";
+	public static String supportTicketsPage = "supportTicketsPage";
+	public static String returnHistoryPage = "returnHistoryPage";
+	public static String shoppingCartPage = "shoppingCartPage";
+	public static String signOutPage = "signOutPage";
+	public static String storeFinderPage = "storeFinderPage";
+	
+
```
Comment (ibatta) [inline src/com/generic/setup/PagesURLs.java]: Please overload the get function below to make it take parameter String driverName, it should use getDriverpath(); where it should return in this case  the full driver path
c:/...../chromeDriver.exe

File: src/com/generic/setup/PagesURLs.java [hunk 2]
```diff
@@ -0,0 +1,222 @@
+package com.generic.setup;
+
+import java.text.MessageFormat;
+import java.util.NoSuchElementException;
+
+import org.apache.poi.hssf.usermodel.examples.NewSheet;
```
Comment (ibatta) [inline src/com/generic/setup/PagesURLs.java]: Please remove this if it is not used

**Comment by isahwahni** [comment]
@ibatta 
The requested changes are done in the following commit : 
https://github.com/ibatta/SelFW/pull/19/commits/050fa1abbacd53837a1f40be21b83442892b4421

Thanks

**Pull request merged**